### DISCUSSION TITLE
Make header in CardHeader in center if no description is provided

### DISCUSSION THREAD

**Discussion by jpeng-ms** [microsoft/fluentui#30688] [General]
Currently, if we don't have a `description`, we still seem to be allocating the space for it: 
<img width="1063" alt="image" src="https://github.com/microsoft/fluentui/assets/109105353/5c2b00fe-0550-4129-9713-9580d4f44043">

<img width="960" alt="image" src="https://github.com/microsoft/fluentui/assets/109105353/36c513b5-0af7-4cb1-85bc-cc1aabac90e2">

I was wondering if we could move `header` to vertical center if description isn't provided? 

https://react.fluentui.dev/?path=/docs/components-card-cardheader--default#default

<img width="936" alt="image" src="https://github.com/microsoft/fluentui/assets/109105353/febeb169-3917-4e87-b7d2-8699b7ff5575">

**Comment by marcosmoura** [comment]
@jpeng-ms This is something that the component specification didn't cover and even though it is not required in code, it always expects a description. I will talk to the designers and see how can this change be made without it being a breaking change.
